docs/api-reference/1_5/documents/_overview.md

```diff
@@ -0,0 +1,92 @@
+# <strong>API OVERVIEW</strong>
+
+Welcome to the Kubernetes API.  You can use our API to access Kubernetes API endpoints to read and write Kubernetes
+resource objects.
+
+## Resource Categories
+
+This is a highlevel overview of the basic types of resources provide by the Kubernetes API and their primary functions.
+
+**Workloads** *Run your containers*: These are responsible for managing and running your containers on the cluster.
```

Comment:
**Workloads** are objects you use to manage and run your containers on the cluster.